CONTRIBUTING.adoc

```diff
@@ -85,7 +85,7 @@ $ sudo dnf install git docker-latest
 ----
 In order to do builds locally, install the following build dependencies:
 ----
-$ sudo dnf install golang golang-race make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito createrepo openssl gpgme gpgme-devel libassuan libassuan-devel
+$ sudo dnf install golang golang-race make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito createrepo openssl gpgme gpgme-devel libassuan libassuan-devel protobuf-compiler
```

Comment:
It's only required if you need to change the API.  You can use:

```
hack/env make update-api
```

to run API generation inside the release docker image (which has protobuf installed already).  You'll need to set the env var to copy that out.